src/main/java/org/spongepowered/api/variant/VariantQueryType.java

```diff
@@ -22,13 +22,18 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package org.spongepowered.api.data.type;
+package org.spongepowered.api.variant;
 
 import org.spongepowered.api.CatalogType;
-import org.spongepowered.api.util.Cycleable;
-import org.spongepowered.api.util.annotation.CatalogedBy;
 
-@CatalogedBy(LogAxes.class)
-public interface LogAxis extends CatalogType, Cycleable<LogAxis> {
+public interface VariantQueryType<T> extends CatalogType {
```

Comment:
What's the point of this? It's feeling very much either like a Property or a Key in some aspects.